sweetest/sweetest-core/src/main/java/com/mysugr/sweetest/usecases/Environment.kt

```diff
@@ -0,0 +1,27 @@
+package com.mysugr.sweetest.usecases
+
+/**
+ * Use cases for the global sweetest environment.
+ *
+ * - achieves internal API stability for calls towards the core while the core can freely be refactored
+ * - adds user-facing exceptions that are shared among different versions of public APIs
+ */
+
+import com.mysugr.sweetest.framework.environment.TestEnvironment
+
+// Temporary means of initializing the environment as long as TestEnvironment is still here
```

Comment:
Comment is no proper KDoc format (`/**`) and I don't know what `Temporary` is referring to 🤔 Maybe it's outdated? (same for the functions below)

Reply:
i used line comments here because it has no documentation function. if it would give more details on the function i would consider it documentation, for example.

but yes, it's still current. when refactoring the core this function will vanish.